client/rest-high-level/src/main/java/org/elasticsearch/client/rollup/GetRollupJobResponse.java

```diff
@@ -51,6 +51,15 @@
     static final ParseField STATE = new ParseField("job_state");
     static final ParseField CURRENT_POSITION = new ParseField("current_position");
     static final ParseField UPGRADED_DOC_ID = new ParseField("upgraded_doc_id");
+    static final ParseField BULK_LATENCY = new ParseField("bulk_latency_in_ms");
+    static final ParseField SEARCH_LATENCY = new ParseField("search_latency_in_ms");
+    static final ParseField SEARCH_FAILURES = new ParseField("search_failures");
+    static final ParseField BULK_FAILURES = new ParseField("bulk_failures");
```

Comment:
Nit: I think it would be nicer to call it `INDEX_FAILURES`, `BULK` is an implementation detail about how indexing is internally implemented.

Reply:
++